types/prettier/index.d.ts

```diff
@@ -266,12 +266,12 @@ export interface SupportLanguage {
     since?: string;
     parsers: BuiltInParserName[] | string[];
     group?: string;
-    tmScope: string;
-    aceMode: string;
-    codemirrorMode: string;
-    codemirrorMimeType: string;
+    tmScope?: string;
+    aceMode?: string;
+    codemirrorMode?: string;
+    codemirrorMimeType?: string;
     aliases?: string[];
-    extensions: string[];
+    extensions?: string[];
     filenames?: string[];
     linguistLanguageId: number;
     vscodeLanguageIds: string[];
```

Comment:
This one is optional, too
```suggestion
    vscodeLanguageIds?: string[];
```